docs/targets/zendesk.md

```diff
@@ -0,0 +1,69 @@
+# Event Target for Zendesk
+
+This event Target recieves [CloudEvents][ce] and interfaces with Zendesk by either creating a ticket or assigning a [tag][zd-tag] on a pre-existing ticket depending on the CloudEvent type.
+
+## Prerequisites
+
+A Zendesk API token is required to utilize this Target. The steps to obtain a token are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type `Zendesk`.
+
+![Adding a Zendesk Target](../images/zendesk-target/create-bridge-1.png)
+
+In the Target creation form, provide a name to the event Target, and add the following information:
+
+* **Default Ticket Subject**: An optional ticket subject fallback if one is not provided in an incoming event
+* **Zendesk Subdomain**: Name of the Zendesk [Subdomain][zd-subdom], without the `zendesk.com` domain or `https://` scheme.
+* **Zendesk Email**: Email address associated with the Zendesk account.
+* **Zendesk API Token**: Reference to a [TriggerMesh secret][tm-secret] containing a [token][zd-token] to communicate with the Zendesk API, as described in the previous section.
+* **Subject**: An optional default ticket subject assignment
```

Comment:
1. Missing dot.
2. I think this is a duplicate, you have two subject fields?

Reply:
yes it is!! sorry about that fixed!